WordPress/src/main/java/org/wordpress/android/ui/activitylog/ActivityLogViewHolder.kt

```diff
@@ -0,0 +1,98 @@
+package org.wordpress.android.ui.activitylog
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageButton
+import android.widget.ImageView
+import android.widget.ProgressBar
+import android.widget.TextView
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.activity.ActivityLogModel
+import java.text.DateFormat
+import java.util.Locale
+
+class ActivityLogViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
+        LayoutInflater.from(parent.context).inflate(R.layout.activity_log_list_item, parent, false)) {
+    private val summary: TextView = itemView.findViewById(R.id.action_summary)
+    private val text: TextView = itemView.findViewById(R.id.action_text)
+    private val thumbnail: ImageView = itemView.findViewById(R.id.action_icon)
+    private val progressBar: ProgressBar = itemView.findViewById(R.id.rewind_progress_bar)
+
+    val header: TextView = itemView.findViewById(R.id.activity_header_text)
+    val button: ImageButton = itemView.findViewById(R.id.rewind_button)
+
+    private var activity: ActivityLogModel? = null
+
+    private val timeFormatter = DateFormat.getDateInstance(DateFormat.LONG, Locale.getDefault())
+
+    init {
+        button.setOnClickListener {
+        }
+    }
+
+    fun bind(activity: ActivityLogModel?) {
```

Comment:
I don't think the parameter should be nullable at this point.

Reply:
Good catch!